themes/beaver/layouts/partials/page/header.html

```diff
@@ -1,123 +1,11 @@
-                                                    <ul class="sub-menu">
-                                                        {{ range .Children }}
-                                                        <li class="submenu-{{ .Identifier }} menu-item menu-item-type-post_type menu-item-object-page {{if (eq $currentPage.RelPermalink .URL) }}current-menu-item page_item current_page_item{{ end }}"><a href="{{ .URL }}">
-                                                            <div class="menu-image">
-                                                                {{ $partialPath := printf "header/icons/%s.html" .Identifier }}
-                                                                {{ partial $partialPath . }}
-                                                            </div>
-                                                            <span class="menu-item-text">{{ .Name }}</span><p class="menu-desc"></p></a></li>
-                                                        {{ end }}
-                                                    </ul>
-                                                </li>
-                                                {{ else }}
-                                                <li class="jt-{{ .Identifier }} menu-item menu-item-type-post_type menu-item-object-page {{if $active }}current-menu-item page_item current_page_item{{ end }}"><a href="{{ .URL }}"><span class="menu-item-text">{{ .Name }}</span></a></li>
-
-                                                {{ end }}
-
-                                                {{ end }}
-                                            </ul>
-                                        </nav>
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                    <div class="fl-col fl-node-header-nav-contact-us-btn fl-col-small fl-visible-desktop fl-visible-large fl-visible-medium" data-node="r5xbj4ni8mge">
-                        <div class="fl-col-content fl-node-content">
-                            <div class="fl-module fl-module-button fl-node-header-nav-contat-us-btn-nested" data-node="vha2g9x6ljec">
-                                <div class="fl-module-content fl-node-content">
-                                    <div class="fl-button-wrap fl-button-width-auto fl-button-right">
-                                        <a href="{{ absURL "contact-us/" }}" target="_self" class="fl-button">
-                                        <span class="fl-button-text">Contact Us</span>
-                                        </a>
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </div>
-        </div>
+<header class="top-panel">
+  <div class="container">
```

Comment:
Is this bootstrap's container?

Reply:
Hard to say. Critical css make unclear that 
Let's rename